### PR TITLE
Add 'syl' example

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

**Comment by expln** [comment]
Hi David. Unfortunately I don't have time to review this PR today. You may merge it or add new commits. I will review tomorrow (even if you merge it). One suspicious thing I noticed in the beginning: you wrote an instruction to move hypotheses to the top. mm-lamp was supposed to move them automatically once you change type P->H. If this didn't happen then this must be a bug. And one more update: I deployed G-statements to dev, but didn't have a chance to update corresponding issue. I will do this tomorrow.

**Comment by david-a-wheeler** [comment]
> Hi David. Unfortunately I don't have time to review this PR today. You may merge it or add new commits. I will review tomorrow (even if you merge it).

No worries! I won't merge it until tomorrow at the earliest.

> One suspicious thing I noticed in the beginning: you wrote an instruction to move hypotheses to the top. mm-lamp was supposed to move them automatically once you change type P->H. If this didn't happen then this must be a bug.

I was doing various experiments, so it's possible that it *did* do that but after doing something else I thought it didn't. I'll re-test it, and remove the text if it's not needed.

I'm trying to make the guide text "flow" in a clear and simple way. However,  to get the examples to that point, I end up experimenting with many approaches. Which means that the description in the guide isn't exactly what I did the first time :-).

> And one more update: I deployed G-statements to dev, but didn't have a chance to update corresponding issue. I will do this tomorrow

Awesome! That will impact various statements in the guide (I think for the better), and it'll be one of the first things that new users will notice. So thanks for putting that in now.

**Comment by david-a-wheeler** [comment]
> One suspicious thing I noticed in the beginning: you wrote an instruction to move hypotheses to the top. mm-lamp was supposed to move them automatically once you change type P->H. If this didn't happen then this must be a bug.

Thanks! I re-tried it and you're right, the statements end up in the right order. I'm not sure how they ended up wrong in my earlier test. I've tweaked the text.

File: docs/index.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ user guide (tutorial) and a reference guide. It includes:
 2. [Sample Screenshot](#sample-screenshot)
 3. [User guide (tutorial)](#user-guide-tutorial)
    shows how to use the metamath-lamp tool, primarily via examples, e.g.,
-   that [2 + 2 = 4 (`2p2e4`)](proof-2--2--4) and the
-   [reciprocal of the cotangent is tangent (`reccot`)](#proof-the-reciprocal-of-the-cotangent-is-tangent-reccot).
+   that [2 + 2 = 4 (`2p2e4`)](proof-2--2--4), the
+   [reciprocal of the cotangent is tangent (`reccot`)](#proof-the-reciprocal-of-the-cotangent-is-tangent-reccot), and the
+   [principle of the syllogism (`syl`)](~proof-principle-of-the-syllogism-syl).
```
Comment (david-a-wheeler) [inline docs/index.md:21]: These 2 line have broken hyperlinks.

**Review by expln — APPROVED** [review]
I reviewed, everything is correct. I will take a look at the unification issue.

**Pull request merged**